scripts/bootstrap.py

```diff
@@ -15,8 +15,14 @@
 import subprocess
 import sys
 import tarfile
-import urllib2
-import urlparse
+
+
+try:  # python 3
+    from urllib.request import Request, urlopen
+    from urllib.parse import urlparse
+except ImportError:  # python 2
+    from urllib2 import Request, urlopen
+    from urlparse import urlparse
```

Comment:
Yeah, this bums me out, but I'm torn about it. I'd originally used six throughout to handle these import issues, but decided to rip it out because there were certain scripts which need to be self contained (this one, and I guess local-deploy-boostrap.py?). My thinking was that ugly consistency was better than inconsistency and that if someone felt strongly the other way, it'd at least be little trouble to go back and add six (or a homegrown compat module) to clean things up where possible.

I'm still on the fence about it and I'll defer to you.